### DISCUSSION TITLE
Is anphon unable to calculate four-phonon processes?

### DISCUSSION THREAD

**Discussion by Lamed-git** [ttadano/alamode#137] [Q&A] [answered]
Dear Dr Ttadano:
I know alamode can calculate fourth order force constants, but I checked the manual and found that anphon seem to be unable to get information about the four phonon processes, such as phase space, scattering rate and so on, is there any reasonable way or example to help me with this? Thank you very much!
Sincerely,
Lamed

**Comment by ttadano** [comment, marked as answer]
The released versions do not support the four-phonon calculations. We will make them available in the next major update (ver. 2.0.0).

**Reply by Lamed-git** [reply]
Thanks for the prompt reply, I wonder when the new version will be released? I see that there is already a 2.0dev version of the branch on github. Also, is it possible to add some features from other branches to the new version? Like FC4_SHENGBTE or something like that. Thank you for developing such an artistic computing program!

**Reply by ttadano** [reply]
Yes, we will merge other features as much as possible into 2.0.

**Reply by Lamed-git** [reply]
Thanks for your generous help!
